doc/HOWTO/keys.txt

```diff
@@ -68,5 +68,42 @@ you don't want your key to be protected by a password, remove the flag
     type in the password every time the server needs to access
     the key.
 
+
+4. To generate an EC key
+
+An EC key can be used both for key agreement (ECDH) and signing (ECDSA).
+
+Generating a key for ECC is similar to generating a DSA key. These are
+two-step processes. First, you have to generate EC parameters from which
+the key will be generated:
+
```

Comment:
So how about use the word 'get', as: 'you have to get the EC parameters from which the key...'